examples/gno.land/p/gnoland/commondao/proposal.gno

```diff
@@ -0,0 +1,174 @@
+package commondao
+
+import (
+	"errors"
+	"std"
+	"time"
+)
+
+// DefaultQuorum defines the default quorum required to tally proposal votes.
+const DefaultQuorum = 0.34 // 34%
+
+const (
+	StatusActive   ProposalStatus = "active"
+	StatusFailed                  = "failed"
+	StatusExecuted                = "executed"
+)
+
+const (
+	ChoiceAbstain VoteChoice = ""
```

Comment:
That might cause problems. If someone wants to abstain, I would define a specific key for that, and fail if the choice is an empty string.

Reply:
Yes, I will take a look at this 👍

Reply:
Addressed in c33540c